Library/Formula/apache-archiva.rb

```diff
@@ -0,0 +1,56 @@
+class ApacheArchiva < Formula
+  desc "The Build Artifact Repository Manager"
+  homepage "https://archiva.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=archiva/2.2.0/binaries/apache-archiva-2.2.0-bin.tar.gz"
+  sha256 "6af7c3c47c35584f729a9c139675a01f9a9819d0cdde292552fc783284a34cfa"
+
+  depends_on :java => "1.7+"
+
+  def install
+    libexec.install Dir["*"]
+
+    bin.install_symlink libexec/"bin/archiva"
+  end
+
+  def post_install
+    (var/"archiva/logs").mkpath
+    (var/"archiva/data").mkpath
+    (var/"archiva/temp").mkpath
+
+    cp_r libexec/"conf", var/"archiva"
```

Comment:
It seems like it'd work okay?

> Copy (move will fail) the configuration files from the Archiva installation (e.g. /opt/archiva-2.0/conf to the new location (e.g. /var/archiva/conf). If you've previously run Archiva, you may need to edit conf/archiva.xml to change the location of the repositories.

> (Optionally, leave a README file in the old conf location as a reminder of where you moved the config files.)
Set the environment variable ARCHIVA_BASE to the new location (e.g. /var/archiva). In bash, be sure to export the variable.

We're passing the variable to the `plist` below, so it'd work, right?